tests/test_trackingstore.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2019 Faculty Science Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+from datetime import datetime
+from uuid import uuid4
+
+import faculty
+from faculty.clients.base import HttpError
+from faculty.clients.experiment import Experiment
+from mlflow.entities import Experiment as MLExperiment, LifecycleStage
+from mlflow.exceptions import MlflowException
+import pytest
+from pytz import UTC
```

Comment:
`pytz` is probably pulled in by mlflow, but if we are using it explicitly we should add it to the tox test dependencies.